libraries/ESP32/examples/Camera/CameraWebServer/CameraWebServer.ino

```diff
@@ -43,6 +37,10 @@ void setup() {
   Serial.setDebugOutput(true);
   Serial.println();
 
+#ifdef CAMERA_MODEL_AI_THINKER
+  WRITE_PERI_REG(RTC_CNTL_BROWN_OUT_REG, 0); //disable brownout detector
+#endif
+
```

Comment:
disabling brownout ... I don't think it is good idea.

Reply:
I can remove or comment out this part if you/Espressif not interested in a that solution. But what you suggest? HW replacement?  Put this module in the trash bin? Make own copy of pre-build idf with disabled BOD??? Rly?

Please let me know about final decision and review other code in this PR too.

Reply:
@me-no-dev, what do you think about it? It is related to #6790 discussion.